proposals/proposal-0001.md

```diff
@@ -0,0 +1,221 @@
+into a project's development workflow with a patch submission to the project's
+issue tracker. The more focused the proposal, the more successful it tends to
+be. As per Helm's governance, the project maintainers reserve the right to
+reject proposal proposals if they appear too unfocused, too broad, or
+incomplete. If in doubt, split your proposal into several well-focused ones, or
+explain your proposal more clearly.
+
+Each proposal must have a champion -- someone who writes the proposal using the
+style and format described below, shepherds the discussions in the appropriate
+communication channels, and attempts to build community consensus around the
+idea. The proposal champion (a.k.a. author) should first attempt to ascertain
+whether the idea is proposal-able. Posting to the helm-users mailing list is the
+best way to go about this.
+
+Vetting an idea publicly before going as far as writing a proposal is meant to
+save the potential author time. Many ideas have been brought forward for
+changing Helm that have been rejected for various reasons. Asking the Helm
+community first if an idea is original helps prevent too much time being spent
+on something that is guaranteed to be rejected based on prior discussions
+(searching the internet does not always do the trick). It also helps to make
+sure the idea is applicable to the entire community and not just the author.
+Just because an idea sounds good to the author does not mean it will work for
+most people in most areas where Helm is used.
+
+Once the champion has asked the Helm community as to whether an idea has any
+chance of acceptance, a summary of the proposal should be presented to
+helm-users. This gives the author a chance to flesh out the proposal as a high
+quality, well-formatted proposal, and to address initial concerns about the
+proposal.
+
+### Submitting a proposal
+
+Following a discussion on helm-users, the proposal should be submitted as a
+proposal via a [GitHub pull request][1]. The proposal must be written in
+proposal style as described below, else it will fail review immediately
+(although minor errors may be corrected by the maintainers).
+
+The standard proposal workflow is:
+
+- You, as the proposal author, fork the helm/community repository and create a
+  new proposal
+- proposals are written in [Markdown][markdown]
+- Push this to your GitHub fork and [submit a pull request][pr].
+
+Once the review process is complete, the project maintainers will approve it
+(note that this is not the same as accepting your proposal!)
+
+The project maintainers will not unreasonably deny a proposal. Reasons for
+denying proposal status include duplication of effort, being technically
+unsound, not providing proper motivation or addressing backwards compatibility,
+or not in keeping with Helm's design. The core maintainers can be consulted
+during the approval phase.
+
+Once approved, proposal authors are responsible for collecting community
```

Comment:
This whole feedback phase feels way too handwave-y and non-specific. We should have a specific way of gathering feedback or it will be a nightmare to manage for maintainers. I personally think a mailing list, no matter how long winded it gets, it probably the least bad option. But we should pick a single option and stick with it

Reply:
How many maintainer approvals are needed to approve a proposal? Questions like this should be answered as part of the documented processes. Both for those submitting and us maintainers.